launcher/0.0.2/launcher-operator.0.0.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,180 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
we need to include the product name in the version-named directory.

i.e. rename directory `0.0.2` to `launcher-0.0.2`